app/views/shelter_pets/index.html.erb

```diff
@@ -1,16 +1,27 @@
 <div class="content">
   <h1>
     <%= link_to(@shelter.name, "/shelters/#{@shelter.id}", class: "name") %> - Pets
-    <%= '(adoptable pets)' if @adoptable %>
-    <%= '(pets that are pending adoption)' if @pending %>
-    (total: <%= @pet_count %>)
+    <% if params[:adoptable] == 'true' %>
+      <%= "(adoptable pets) (total: #{@shelter.pet_count("Adoptable")})" %>
+    <% elsif params[:adoptable] == 'false' %>
+      <%= "(pets pending adoption) (total: #{@shelter.pet_count("Pending")})" %>
+    <% else %>
+      <%= "(total: #{@shelter.pet_count})" %>
+    <% end %>
```

Comment:
same idea here, except the shelter pets index also displays the relevant pet count.